include/deal.II/dofs/dof_accessor.templates.h

```diff
@@ -3075,8 +3075,10 @@ namespace internal
         (void)i;
         // ::DoFHandler only supports a single active fe with index
         // zero
+#ifdef DEBUG
         typedef dealii::DoFAccessor<dim,DoFHandler<dim,spacedim>, level_dof_access> BaseClass;
         Assert (i == 0, typename BaseClass::ExcInvalidObject());
+#endif
```

Comment:
The fact that back in 1998 I didn't understand why the parentheses were necessary is why the typedef is there :-(